packages/frontend/src/components/Routing.js

```diff
@@ -369,10 +357,10 @@ class Routing extends Component {
                 >
                     <ThemeProvider theme={theme}>
                         <ScrollToTop />
-                        <NetworkBanner account={account} />
-                        <NavigationWrapper history={this.props.history}/>
+                        {pathname !== '/' && <NetworkBanner account={account} />}
+                        {pathname !== '/' && <NavigationWrapper history={this.props.history}/> }
                         <GlobalAlert />
-                        {
+                        {/* {
```

Comment:
Please delete all commented-out routes so that it is clear exactly which routes remain.